public/index.html

```diff
@@ -9,8 +9,8 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-	<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+	<!--	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">-->
+	<!--	<link rel="shortcut icon" href="%PUBLIC_URL%/fav/favicon.ico">-->
```

Comment:
```suggestion
	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
	<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
```

Reply:
But where is the manifest?

Reply:
Ooops :grin:  You are right, there is no manifest, let's remove this line.